### ISSUE TITLE
Bug: Scanning a QR code with Zeus leads to an error - no information about further steps available

### ISSUE THREAD

**Issue by AnastasiaVolk** [getAlby/lightning-browser-extension#1129]
**Bug description**
Scanning a QR code with Zeus leads to an error - no information about further steps available

**Screenshots** 
<img width="1272" alt="The power of lightning in your browser" src="https://user-images.githubusercontent.com/108762093/178040924-90b93bcd-d39f-4d4b-b524-b1fecbc277e3.png">

**Steps to reproduce the behaviour:**

1. Visit to getalby.com
2. Click on "Install Alby"
3. Click "Add extension to Chrome"
4. Click "Get started"
5. Input a password
6. Confirm a password
7. Click "Next"
8. Input new email
9. Input correct password
10. Click “Continue”
11. Scan the QR code with Zeus

**Expected behaviour**
Should be information about possible further actions

Alby information

* Alby Version: [v1.11.2]
* Alby installed through Google Chrome

Device information

- OS: [macOS Monterey 12.0.1 ] 
- Browser [chrome, safari]  Browser Version [Safari Version 15.1, Chrome Version 103.0.5060.114]

Are you working on this issue?
No

**Comment by escapedcat** [comment]
Is this related or a duplicate of #1128 ?

**Comment by bumi** [comment]
no, this seems about adding the wallet to Zeus. We should explain better what the user can do there. And we should also hide the QR code by default as it contains sensitive data (@AnastasiaVolk I will create a new wallet for you)

**Comment by AnastasiaVolk** [comment]
@bumi thanks, that would be very cool! As I already had to create a couple of them for testing😅 and yes you are super right about the bug understanding!
